tic_tac_toe.py

```diff
@@ -0,0 +1,111 @@
+import random
+
+
+class TicTacToe:
+
+    def __init__(self):
+        self.board = [None]*9
+
+    def make_move(self, row: int, col: int, player):
+        position = col*3 + row
+        self.board[position] = player
```

Comment:
Need to check wether the cell is already occupied before assigning the position. Otherwise user can overwrite  a cell that have already been marked.